deployment/terraform/main.tf

```diff
@@ -0,0 +1,356 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "<= 5.72.1"
+    }
+  }
+}
+
+provider "aws" {
+  region                     = "us-east-1"
+  skip_requesting_account_id = var.local_run
+}
+
+variable "local_run" {
+  description = "Flag to determine if running locally using LocalStack"
+  type        = bool
+}
+
+resource "random_integer" "images_bucket_id" {
+  min = 0
+  max = 10000
+}
+
+resource "random_integer" "image_resized_bucket_id" {
+  min = 0
+  max = 10000
+}
+
+resource "random_integer" "website_bucket_id" {
+  min = 0
+  max = 10000
```

Comment:
nit: we can rather use a variable to define `min` and `max` for above three resources as they share the same static values.

Reply:
Thanks for the nit, but this change is not needed anymore. I removed random numbers for bucket names to keep the code cleaner after removal of `local_run` variable.